resources/packs/azure/azure.lr

```diff
@@ -108,13 +108,13 @@ azuread.serviceprincipal {
   id string
 }
 
-// Azure Resource Manager
+// Azure Resource
```

Comment:
This description is now the same as L117, but they are different resources.

Perhaps just add an "s" here to distinguish them?

Reply:
Fixed